sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/repository/SecondaryTestRepositoryConfig.java

```diff
@@ -34,6 +34,9 @@ public class SecondaryTestRepositoryConfig {
     @Value("${cosmos.secondary.queryMetricsEnabled}")
     private boolean queryMetricsEnabled;
 
+    @Value("${cosmos.secondary.maxDegreeOfParallelism:0}")
```

Comment:
Somewhere I thought I saw that this is defining a default value if it is not supplied, but now I can't find where I thought that (it may have been a check in the PR that wanted a default). I can remove if we don't want it.

Reply:
Yes, we should remove it.

Reply:
Done.